src/griddyn/loads/approximatingLoad.cpp

```diff
@@ -537,7 +531,7 @@ std::vector<double> approximatingLoad::run3ApproxB ()
 }
 
 return retP;
-}
+}  // namespace loads
```

Comment:
this doesn't seem like a namespace closure, it is just a function termination

Reply:
Oh oops